### PR TITLE
Update example in README.md

### PR DESCRIPTION
**Pull request by linyows**
I tried this method with ubuntu/xenial and was able to launch haconiwa cotainer.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,15 +46,22 @@ If you would not, these installation are not required.
...
+
+```sh
+$ mkdir -p /var/another/root/etc
+$ mkdir -p /var/another/root/home
+```
+
 Create the file `example.haco`:
 
 ```ruby
-Haconiwa::Base.define do |config|
```
Comment (linyows) [inline README.md]: Got error as `Direct call of Haconiwa::Base.define is deprecated. Please rewrite into Haconiwa.define (RuntimeError)`

File: README.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ Available for: `CentOS >= 7 / CentOS ~> 6(Experimental, maybe kernel update requ
 Other linuxes users can just download binaries from [latest](https://github.com/haconiwa/haconiwa/releases):
 
 ```bash
-VERSION=0.8.0
+VERSION=<specify version>
```
Comment (linyows) [inline README.md]: v0.8.0 is outdated, now

File: README.md [hunk 3]
```diff
@@ -46,15 +46,22 @@ If you would not, these installation are not required.
...
+```sh
+$ mkdir -p /var/another/root/etc
+$ mkdir -p /var/another/root/home
+```
+
 Create the file `example.haco`:
 
 ```ruby
-Haconiwa::Base.define do |config|
+Haconiwa::Base do |config|
```
Comment (udzura) [inline README.md]: So, now i use `Haconiwa.define`
https://github.com/haconiwa/haconiwa/blob/af3d22c822baea165bd825cda0f04f39c25bbc7e/mrblib/haconiwa/generator.rb#L5

Comment (linyows) [reply README.md]: Oh, I mistook the correction...

**Comment by udzura** [comment]
Thanks!

**Pull request merged**